Core/include/Acts/Seeding/BinnedSPGroup.ipp

```diff
@@ -74,7 +74,18 @@ Acts::BinnedSPGroupIterator<external_spacepoint_t>::operator*() const {
           m_group->m_bins[m_current_localBins[INDEX::Z]],
           m_group->m_grid.get());
 
+  // GCC12+ in Release throws an overread warning here due to the move.
+  // This is from inside boost code, so best we can do is to suppress it.
+#if defined(__GNUC__) && !defined(__clang__)
```

Comment:
why not

```suggestion
#if defined(__GNUC__) && __GNUC__ >= 11 && !defined(__clang__)
```
and omitting the second `#if`?

Reply:
Can do!

Reply:
Done now.